components/celplace/src/main/java/com/celements/crm/place/geocoding/GeocodingService.java

```diff
@@ -56,9 +59,27 @@ GeoApiContext getGMapsContext() {
     return gMapsContext;
   }
 
+  @Override
+  public ClassDefinition getGeotagClass() {
+    String geotagClassName = configSource.getProperty(
+        "celements.geocoding.celementsPlaces.geotagClassName", String.class);
+    ClassDefinition classDef;
+    if (!Strings.isNullOrEmpty(geotagClassName)) {
+      if (Utils.getComponentManager().hasComponent(ClassDefinition.class, geotagClassName)) {
+        classDef = Utils.getComponent(ClassDefinition.class, geotagClassName);
+      } else {
+        classDef = Utils.getComponent(ClassDefinition.class,
+            IGeocodingServiceRole.CRM_PLACE_DEFAULT_GEOTAG_CLASSNAME);
+      }
```

Comment:
`DRY`
-  only one `if` is needed, you can check both conditions there.
- only call `Utils.getComponent` once if you know the valid `componentName`

Reply:
**not addressed**